charts/nd-common/templates/_datadog.tpl

```diff
@@ -28,6 +28,24 @@ ad.datadoghq.com/{{ .Chart.Name }}.instances: |-
   ]
 {{- end }}
 
+{{/*
+This is datalog logging configuration. We take the .Values.scrapeLogs and
+.Values.scrapeLogsProcessingRules map and convert into list of objects converted
+into json supported by datadog config. If source and service tag values not provided
+we add default values to it.
+*/}}
+
+{{- if and .Values.datadog.enabled .Values.datadog.scrapeLogs.enabled }}
+ad.datadoghq.com/{{ .Chart.Name }}.logs: |-
```

Comment:
Do you want to create a `nd-common.containerName` function in this PR while you are here? It'll be pretty simple:

```
# _common.tpl
{{- define "nd-common.containerName" -}}
{{ .Chart.Name }}
{{- end }}
```

but that will make it easier down the road for us to add customization.. for example
```

# _common.tpl
{{- define "nd-common.containerName" -}}
{{ default .Chart.Name .Values.containerName }}
{{- end }}
```